<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wizard</class>
 <widget class="QWizard" name="wizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Datasource Conversion</string>
  </property>
  <widget class="QWizardPage" name="inputWizardPage">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="DatasourceManagementWidget" name="datasourceManagementWidgetIn" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="outputWizardPage">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="DatasourceManagementWidget" name="datasourceManagementWidgetOut" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="mappingWizardPage">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="refreshPushButton">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../../resources.qrc">
        <normaloff>:/plugins/DsgTools/icons/refresh.png</normaloff>:/plugins/DsgTools/icons/refresh.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DatasourceManagementWidget</class>
   <extends>QWidget</extends>
   <header>DsgTools.gui.CustomWidgets.DatabaseConversionWidgets.datasourceManagementWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
